source/manual/govuk-notify.html.md

```diff
@@ -81,12 +86,6 @@ add yourself to the service:
 2. The members with the permission `Manage settings, team and usage` will be
    able to add you to this team.
 
-**Note:** some of our applications, notably Email Alert API, has an extra level
-of protection and [there is an extra step][email-alert-api-receive-emails]
-before you can receive emails through Email Alert API.
-
-[email-alert-api-receive-emails]: /manual/receiving-emails-from-email-alert-api-in-integration-and-staging.html
-
 The GOV.UK Publishing services are a special case: for these services,
```

Comment:
I'm not too sure what platform health have set-up for their various apps. I think it might be just Content Publisher and Email Alert API that use groups. I did wince a bit looking at this section but didn't want to extend the scope further (hence the cop-out note at the top).

Reply:
Sure, fair enough. That doc goes beyond the email system.